### PR TITLE
policy: Enable controller logs at INFO level

### PR DESCRIPTION
**Pull request by olix0r**
Dependencies like `kubert` may emit INFO level logs that are useful to
see (e.g., when the serviceaccount has insufficient RBAC). This change
updates the default policy controller log level to simply be `info`.

Signed-off-by: Oliver Gould <ver@buoyant.io>

### CODE REVIEW

**Review by hawkw — APPROVED** [review]
Looks like the chart readmes and golden files need to be regenerated? Other than that, LGTM when CI passes!

**Review by adleong — APPROVED** [review]

**Pull request merged**
